server/src/test/java/org/candlepin/test/TestUtil.java

```diff
@@ -437,6 +437,31 @@ public static Date createDate(int year, int month, int day) {
         return cal.getTime();
     }
 
+    public static Date futureDateInYears(int addedYears) {
+        Calendar cal = Calendar.getInstance();
+
+        cal.add(Calendar.YEAR, addedYears);
+
+        return cal.getTime();
+    }
+
+    public static Date pastDateInYears(int removedYears) {
+        Calendar cal = Calendar.getInstance();
+
+        cal.add(Calendar.SECOND, -removedYears);
+
+        return cal.getTime();
+    }
+
+    public static Date roundDownToSeconds(Date date) {
+        Calendar cal = Calendar.getInstance();
```

Comment:
It looks like java.time.Instant has nifty rounding-down methods, so I added that to the Util class and used it both in the production (instead of apache commons lib) & tests